src/managers/GuildMemberManager.js

```diff
@@ -224,7 +225,14 @@ class GuildMemberManager extends BaseManager {
       .then(data => this.add(data, cache));
   }
 
-  _fetchMany({ limit = 0, withPresences: presences = false, user: user_ids, query, time = 120e3 } = {}) {
+  _fetchMany({
+    limit = 0,
+    withPresences: presences = false,
+    user: user_ids,
+    query,
+    time = 120e3,
+    nonce = this.guild.id,
```

Comment:
Well, I have another suggestion, consider generating a value differently instead of using the guild ID as the default nonce. Something like `Date.now().toString(36)` should be simple enough

---

Why? Consider these two calls: `members.fetch({ query: 'spa', limit: 2 });` and `members.fetch({ query: 'vla', limit: 2 });`; they are two different queries but would use the same nonce. Thus, results for `spa` could return the results for `vla` or vice versa.